src/components/Rockets.jsx

```diff
@@ -0,0 +1,35 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
@Team it would have been nice if you tried and solve the linter error instead of disabling the linter checks on your file, it's advisable to use the linter diable as a last resort